responsive.css

```diff
@@ -0,0 +1,450 @@
+        rgba(255, 67, 34, 1) 100%
+      );
+    width: 60px;
+  }
+
+  .article-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    margin: 0 auto;
+    grid-gap: 20px;
+    width: 85%;
+  }
+
+  .news-pages {
+    margin: 25px 50px;
+  }
+
+  .news-container {
+    max-width: 300px;
+    margin: 0 auto;
+  }
+
+  .latest-news {
+    margin: 20px auto;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 15px;
+  }
+
+  .article2 {
+    display: flex;
+    justify-content: space-between;
+    margin: 100px auto 0 auto;
+    width: 90%;
+  }
+
+  .ul3 p {
+    padding-top: 8px;
+    color: rgb(34, 34, 34);
+    font-size: 18px;
+  }
+
+  .ul4 li p {
+    padding-top: 8px;
+    color: rgb(34, 34, 34);
+    font-size: 18px;
+  }
+
+  .one {
+    padding-right: 13px;
+    border-right: 1.5px solid rgb(241, 106, 57);
+  }
+
+  .two {
+    padding-right: 10px;
+    border-right: 1.5px solid rgb(241, 106, 57);
+    padding-left: 13px;
+  }
+
+  .three {
+    padding-left: 13px;
+  }
+
+  .more-list {
+    margin: 25px 50px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+  }
+
+  .more-list .header-img {
+    width: 100%;
+  }
+
+  .hardfork-img .hardfork-content {
+    position: absolute;
+    bottom: 8%;
+    display: flex;
+    flex-direction: column;
+    font-size: small;
+    padding-left: 10px;
+    color: white;
+  }
+
+  .footer2 {
+    display: flex;
+    justify-content: center;
+    background-color: #000;
+    color: #9d9999;
+    padding: 50px;
+  }
+}
+
+@media only screen and (max-width: 1024px) and (min-width: 769px) {
+  #tnw-main-article-image {
+    display: none;
+  }
+
+  .latest-news {
+    margin: 20px auto;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .medium-nav {
+    display: grid;
+    grid-template-columns: 101px 1fr;
+    grid-auto-rows: 1fr 1fr;
+    grid-template-areas:
+      "mobile-nav2 nav1"
+      "mobile-nav2 nav2";
+  }
+
+  .mobile-nav2 {
+    background: rgb(210, 18, 137);
+    background:
+      linear-gradient(
+        45deg,
+        rgba(210, 18, 137, 1) 0%,
+        rgba(255, 67, 34, 1) 100%
+      );
+    width: 100px;
+    height: 120px;
+    grid-area: mobile-nav2;
+  }
+
+  .mobile-nav2 img {
+    padding-top: 40px;
+  }
+
+  .nav2 {
+    background-color: #fff;
+    color: #aaa;
+    grid-area: nav2;
+    display: flex;
+    justify-content: space-between;
+    font-weight: 500;
+    font-size: 15px;
+    height: 100%;
+  }
+
+  .nav1 {
+    background-color: #131313;
+    color: #9f9f9f;
+    grid-area: nav1;
+    display: flex;
+    justify-content: space-between;
+    font-size: 12px;
+    height: 100%;
+  }
+
+  .nav1 ul,
+  .nav2 ul {
+    list-style: none;
+    display: flex;
+  }
+
+  .section1 {
+    padding-top: 100px;
+  }
+
+  .article2 {
+    display: flex;
+    flex-direction: column;
+    margin-left: 50px;
+  }
+
+  .article-container-4 {
+    display: none;
+  }
```

Comment:
Same style definition of the selector on line 129. Group selectors, or create a new class.